keccak-air/examples/prove_baby_bear_poseidon2.rs

```diff
@@ -68,8 +68,8 @@ fn main() -> Result<(), impl Debug> {
     let trace = generate_trace_rows::<Val>(inputs);
 
     let fri_config = FriConfig {
-        log_blowup: 1,
-        num_queries: 100,
+        log_blowup: 3,
+        num_queries: 33,
```

Comment:
I'd slightly prefer keeping the old config, mainly to match the other examples for easy comparison. If examples with other configs like this are useful, maybe we could generalize the examples to parse CLI some options.

Reply:
whoops, I didn't even realize I'd changed it for some other testing